nativescript-slides.ts

```diff
@@ -9,6 +9,8 @@ import * as AnimationModule from 'ui/animation';
 import * as gestures from 'ui/gestures';
 import {AnimationCurve} from 'ui/enums';
 
+const LayoutParams = android.view.WindowManager.LayoutParams;
+
```

Comment:
It also won't have clearFlags() that was after android 17

On Fri, May 20, 2016, 6:37 AM Brad Martin <bradwaynemartin@gmail.com> wrote:

> Should have it. But it won't have all the flags since it's API 17 and
> these are 19.
>
> On Fri, May 20, 2016, 5:56 AM Josh Sommer <notifications@github.com>
> wrote:
>
>> In nativescript-slides.ts
>> <https://github.com/TheOriginalJosh/nativescript-slides/pull/9#discussion_r64023394>
>> :
>>
>> > @@ -9,6 +9,8 @@ import * as AnimationModule from 'ui/animation';
>> >  import * as gestures from 'ui/gestures';
>> >  import {AnimationCurve} from 'ui/enums';
>> >
>> > +const LayoutParams = android.view.WindowManager.LayoutParams;
>> > +
>>
>> my android d.ts file doesn't have layoutParams, did you get yours from
>> tns-platform-declarations... I think i'm behind.
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly or view it on GitHub
>> <https://github.com/TheOriginalJosh/nativescript-slides/pull/9/files/099adf0c594cdfe550436750e558ced1a94f81c1#r64023394>
>>
>